src/pybind/mgr/ansible/run-tox.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # run from ./ or from ../
-: ${MGR_ANSIBLE_VIRTUALENV:=/tmp/mgr-ansible-virtualenv}
```

Comment:
actually, make check is supposed to set MGR_ANSIBLE_VIRTUALENV . the right side of `:=` is only meant to be the default value.

Reply:
@sebastian-philipp i knew. and i think i am setting the default value here. are you suggesting something different?

Reply:
can we set the default to `$current_script_dir/../../../build/` or something similar?

Reply:
could you help me understand why `$current_script_dir/../../../build/` is better than `$CEPH_BUILD_DIR` ?

Reply:
Because `run-tox.sh` can be used outside of `make test`: it can be used by the developer to quickly run tox without needing to execute `make check`.  It would be a tiny bit easier to not having to manually pass CEPH_BUILD_DIR to `tun-tox`

Reply:
@sebastian-philipp is `ctest -R run-tox-mgr-ansible` an alternative for you?

Reply:
Needs to be communicated to @LenzGr 's team

Reply:
sure, i am adding @LenzGr for his team's inputs.